test/mocha.opts

```diff
@@ -1,4 +1,5 @@
 --require babel-register
+--require jsdom-global/register
```

Comment:
It is needed for reference to `instanceof HTMLElement` usage in code, under mocha, `HTMLElement` is `undefined`.